ui/src/custom/scanMetaSchemas.ts

```diff
@@ -13,16 +13,16 @@ export interface AppMeta {
 }
 
 // validation schemas...
-const scanMetaSchema: Yup.SchemaOf<ScanMeta> = Yup.object()
+const scanMetaSchema: Yup.ObjectSchema<ScanMeta> = Yup.object()
 	.shape({
 		// these can't be required (.defined()) because entire schema object is optional (see interface)
```

Comment:
Does this comment still apply? I don't know anything about the library, but this PR seems to add `.defined()`

Reply:
agreed, this comment is confusing. I have removed it